src/i18n/strings/en_US.json

```diff
@@ -128,6 +128,7 @@
     "Kick": "Kick",
     "Kicks user with given id": "Kicks user with given id",
     "Labs": "Labs",
+    "LaTeX math in messages": "LaTeX math in messages",
```

Comment:
Please revert this translation here, as it can cause conflicts. Once the main PR has merged, you can submit this translation via https://translate.element.io.